packages/dd-trace/src/telemetry/send-data.js

```diff
@@ -1,14 +1,24 @@
 const request = require('../exporters/common/request')
 let seqId = 0
+
+function trimPayload (payload) {
+  const { logger, tags, serviceMapping, ...trimmedPayload } = payload
```

Comment:
We are assuming that `payload` object will always contain `logger`, `tags`, `serviceMapping` properties, otherwise this will fail.